specification/ml/put_job/MlPutJobRequest.ts

```diff
@@ -41,12 +42,14 @@ export interface Request extends RequestBase {
     analysis_limits?: AnalysisLimits
     background_persist_interval: Time
     custom_settings?: CustomSettings
-    data_description?: DataDescription
     daily_model_snapshot_retention_after_days?: long
-    groups?: string[]
+    data_description?: DataDescription
```

Comment:
I think this is required

Reply:
Thanks!  I've changed that in 55c4246a4b20282f5cad9cd1c5e5d3110b0ed227